samples/Owin.Nowin.HelloWorld/Owin.Nowin.HelloWorld.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <Import Project="..\..\build\dependencies.props" />
+
+  <PropertyGroup>
+    <IsPackable>false</IsPackable>
+    <TargetFramework>net461</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Hosting" Version="$(AspNetCoreVersion)" />
+    <PackageReference Include="Microsoft.AspNetCore.Owin" Version="$(AspNetCoreVersion)" />
+    <PackageReference Include="Microsoft.AspNetCore.Server.IISIntegration" Version="$(AspNetCoreVersion)" />
+    <PackageReference Include="Nowin" Version="$(NowinVersion)" />
+  </ItemGroup>
+
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net461' ">
```

Comment:
I thought it was better to keep this PackageReference behind a condition, for a few reasons.  First, it's more consistent with the majority of samples which target `netcoreapp2.0;net461`.  Second, if we ever did decide to add `netcoreapp2.0` to this project, it would be incorrect to unconditionally include this PackageReference.  But if we're **not** putting this PackageReference behind a condition in other samples, I can do the same for Entropy.